package.json

```
@@ -9,6 +9,7 @@
 		"lint": "eslint src",
 		"types-check": "tsc --noEmit",
 		"build": "npx esbuild --bundle --platform=node ./src/index.ts --external:esbuild --external:chokidar --outfile=./dist/index.js",
+		"watch": "npm run build -- --watch",
```

```suggestion
		"build:watch": "npm run build -- --watch",
```
or do you find it too verbose? 🤔